src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7) {
```

Comment:
The condition checks for exactly 7 days, which is not correct. The discount should apply for 7 or more days. Also, avoid using a hard-coded return value.